gp-templates/helper-functions.php

```diff
@@ -213,20 +213,18 @@ function map_glossary_entries_to_translation_originals( $translation, $glossary
 		}
 
 		// Build the regular expression.
-		$terms_search = '\b(';
+		$terms_search = '((?<!%)%(?:[1-9]\$)?[bcdefgosuxEFGX%l@])|(\b';
```

Comment:
For me the test also passes like this, can we remove it?
```suggestion
		$terms_search = '(%(?:[1-9]\$)?[bcdefgosuxEFGX%l@])|(\b';
```

Reply:
I have added a new check in the current tests to address this question and this part is not needed, so I have committed this improvement.